src/pymordemos/neural_networks.py

```diff
@@ -65,23 +64,60 @@ def main(
         fom.visualize((U, U_red),
                       legend=('Full solution', 'Reduced solution'))
 
+    output_reductor = NeuralNetworkOutputReductor(fom, training_set, validation_set, validation_loss=1e-5)
+    output_rom = output_reductor.reduce(restarts=100)
+
+    outputs = []
+    outputs_red = []
+    outputs_speedups = []
+
+    print(f'Performing test on set of size {len(test_set)} ...')
+
+    for mu in test_set:
+        tic = time.perf_counter()
+        outputs.append(fom.compute(output=True, mu=mu)['output'])
+        time_fom = time.perf_counter() - tic
+
+        tic = time.perf_counter()
+        outputs_red.append(output_rom.compute(output=True, mu=mu)['output'])
+        time_red = time.perf_counter() - tic
+
+        outputs_speedups.append(time_fom / time_red)
+
+    outputs = np.squeeze(np.array(outputs))
+    outputs_red = np.squeeze(np.array(outputs_red))
+
+    outputs_absolute_errors = np.abs(outputs - outputs_red)
+    outputs_relative_errors = np.abs(outputs - outputs_red) / np.abs(outputs)
```

Comment:
It would be nice to also compute the output errors for the older ANN ROM and compare the results. It would be great if you could extend the tutorial as well.

Reply:
The first part is done in cf35635, the second in b4c14a4. Maybe you can take a short look at the tutorial if that is enough on the new reductors, @sdrave? I also included a short section on the instationary reductors, since that was missing so far as well.